weblate/accounts/tests/test_notifications.py

```diff
@@ -342,6 +344,31 @@ def test_notify_new_announcement(self) -> None:
             "[Weblate] New announcement at Weblate",
         )
 
+    def test_notify_component_translated(self) -> None:
+        unit = self.get_unit()
+        unit.change_set.create(
+            user=self.anotheruser,
+            old="",
+            action=Change.ACTION_COMPLETED_COMPONENT,
+        )
+
+        # Check mail - TranslatedComponentNotification
+        self.validate_notifications(
+            1, "[Weblate] All language translations have been completed in Test/Test"
+        )
+
+    def test_notify_language_translated(self) -> None:
+        unit = self.get_unit(language="cs")
+        unit.change_set.create(
+            user=self.anotheruser,
+            details={"language": "de"},
+            action=Change.ACTION_COMPLETE,
+        )
```

Comment:
Please create change in the same scope as in the real code:

```suggestion
        unit.translation.change_set.create(
            user=self.anotheruser,
            action=Change.ACTION_COMPLETE,
        )
```